src/components/form/form.stories.mdx

```diff
@@ -72,6 +72,12 @@ When either `errorCount` or `warningCount` or both are provided, summary with th
   <Story name="with errors summary" story={stories.WithErrorsSummary} />
 </Canvas>
 
+When `required` and `isOptional` are provided.
```

Comment:
Comment: we shouldn't have this story as when the invariant is updated it will throw and all we'll see is an error etc